Automattic/wp-calypso
# Videos UI: Removing unused stylesheet

#### Changes proposed in this Pull Request

* This PR removes `style-video-bar.scss`, which was no longer being used after recent refactors to the Videos UI and could be safely deleted.

#### Testing instructions



* Check out this PR locally.
* Verify that the Videos UI still displays without any style changes; this PR should not cause any user-facing updates.



Fixes #58478
